tests/basic/kubevirt_test.go

```diff
@@ -827,3 +830,86 @@ var _ = Describe("{KubeVirtPvcAndPoolExpandWithAutopilot}", func() {
 		}
 	})
 })
+
+var _ = Describe("{RebootRootDiskAttachedNode}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("RebootRootDiskAttachedNode", "Reboot the node where VMs root disk is attached", nil, 0)
+		DeployVMTemplatesAndValidate()
+	})
+	var appCtxs []*scheduler.Context
+
+	itLog := "Reboot node where Kubevirt VMs root disk is attached"
+	It(itLog, func() {
+		appList := Inst().AppList
+		defer func() {
+			Inst().AppList = appList
+		}()
+		Inst().AppList = []string{"kubevirt-cirros-live-migration", "kubevirt-windows-vm"}
+		stepLog := "schedule a kubevirtVM"
+		Step(stepLog, func() {
+			for i := 0; i < Inst().GlobalScaleFactor; i++ {
+				log.Infof("RK=> Deploying iteration: %d", i+1)
+				appCtxs = append(appCtxs, ScheduleApplications("reboot")...)
+			}
+		})
+		//defer DestroyApps(appCtxs, nil)
+		ValidateApplications(appCtxs)
+		for _, appCtx := range appCtxs {
+			bindMount, err := IsVMBindMounted(appCtx, false)
+			log.FailOnError(err, "Failed to verify bind mount")
+			dash.VerifyFatal(bindMount, true, "Failed to verify bind mount")
+		}
+
+		stepLog = "Get node where VM's root disk is attached and reboot that node"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+
+			for _, virtualMachineCtx := range appCtxs {
```

Comment:
It would be better to iterate over the nodes instead of the VMs. That way, we will reboot each node only once and verify all VMs in one go for scalability.

Reply:
@pureneelesh based on our conversation I feel, the test should do it with rebooting nodes with VMs on it, instead of iteration over nodes. This way the test will introduce more chaos, as expected of a system test.
Also we can add another test in stork functional test similar to `SimulateOCPUpgrade` test that does a reboot of node that will cover the scenario suggested by you.